northstar/src/util.rs

```diff
@@ -39,3 +39,17 @@ pub(crate) async fn is_rw(path: &Path) -> bool {
         Err(_) => false,
     }
 }
+
+pub(crate) trait PathExt {
+    fn join_strip<T: AsRef<Path>>(&self, w: T) -> PathBuf;
+}
+
+impl PathExt for Path {
+    fn join_strip<T: AsRef<Path>>(&self, w: T) -> PathBuf {
+        if w.as_ref().starts_with("/") {
+            self.join(w.as_ref().strip_prefix("/").unwrap())
+        } else {
+            self.join(w)
+        }
```

Comment:
this would avoid the `unwrap`
```
        self.join(match w.as_ref().strip_prefix("/") {
            Some(stripped) => stripped,
            None => w,
        })
```

Reply:
Fixed.